### PR TITLE
Remove the python dependency

### PR DESCRIPTION
**Pull request by Kcchouette**
Be careful, I've committed this file using encoding to UTF-8 (not as ISO-8859-1)

### CODE REVIEW

File: examples/certs/make_keyprint.sh [hunk 1]
```diff
@@ -2,21 +2,21 @@
 #
 # make_keyprint.sh
 #
-# based on a script by ullner
-# source: https://adc.svn.sourceforge.net/svnroot/adc/trunk/Source/scripts/keyp_keyprint_generation.sh
```
Comment (Kcchouette) [inline examples/certs/make_keyprint.sh:6]: Moreover the new source is https://gitlab.com/dcnf/adc-code/blob/master/Source/scripts/keyp_keyprint_generation.sh

**Comment by Kcchouette** [comment]
ping @Sopor @pulsar-de

**Comment by Kcchouette** [comment]
ping @Sopor @pulsar-de

**Comment by blasti** [comment]
Thanks for your effort, but it is not required to use an external script anymore for generating the keyprint.

I introduced ` etc_keyprint.lua` in bbe89b4bd793cb1930ccc68a7793c16742b6543a, which solves the problem automatically.

I will leave the script for historical reasons though.

**Pull request closed without merging**